spacy/strings.pyx

```diff
@@ -174,30 +201,29 @@ cdef class StringStore:
         """
         return self.keys.size()
 
-    def __contains__(self, string not None):
-        """Check whether a string is in the store.
+    def __contains__(self, string_or_id not None):
+        """Check whether a string or ID is in the store.
 
-        string (str): The string to check.
+        string_or_id (str or int): The string to check.
         RETURNS (bool): Whether the store contains the string.
         """
-        cdef hash_t key
-        if isinstance(string, int) or isinstance(string, long):
-            if string == 0:
+        cdef hash_t id
+        if isinstance(string_or_id, str):
+            if len(string_or_id) == 0:
                 return True
-            key = string
-        elif len(string) == 0:
-            return True
-        elif string in SYMBOLS_BY_STR:
-            return True
-        elif isinstance(string, str):
-            key = hash_string(string)
+            elif string_or_id in SYMBOLS_BY_STR:
+                return True
+            id = hash_string(string_or_id)
+        elif try_coerce_to_hash(string_or_id, &id):
+            pass
         else:
-            string = string.encode("utf8")
-            key = hash_utf8(string, len(string))
-        if key < len(SYMBOLS_BY_INT):
+            # TODO: Raise an error instead
```

Comment:
Some question here, is it a TODO because otherwise the API would change?

Reply:
Correct.